downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -50,8 +50,34 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 ### New features
 
-#### `avro` package
+#### `persist` package
+- Incorporated support for the PostgreSQL data store, mirroring the functionality provided for other supported SQL data stores like MySQL and MSSQL.
```

Comment:
Maybe
```suggestion
- Introduced support for the PostgreSQL data store, mirroring the functionality provided for other supported SQL data stores like MySQL and MSSQL.
```

Reply:
Fixed in [70ca3c7](https://github.com/ballerina-platform/ballerina-dev-website/pull/8881/commits/70ca3c7b388d2526aa364810f78adc5a3f47dd67)